external/java/src/main/java/ch/epfl/dedis/lib/eventlog/Event.java

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.dedis.lib.eventlog;
+
+import ch.epfl.dedis.proto.EventLogProto;
+
+/**
+ * An instance of an Event can be sent and stored by the EventLog service.
+ */
+public class Event {
+    private long when; // in nano seconds
+    private String topic;
+    private String content;
+
+    public Event(long when, String topic, String content) {
```

Comment:
This really needs some documentation - at least to tell that `long` is in nanoseconds or so.